markets/bfp-market/test/integration/modules/PerpAccountModule.mergeAccounts.test.ts

```diff
@@ -300,55 +319,36 @@ describe('PerpAccountModule mergeAccounts', () => {
     );
     const order = await genOrder(bs, market, collateral, collateralDepositAmount, {
       desiredLeverage: 1,
+      desiredHooks: [MergeAccountSettlementHookMock.address],
     });
     await commitOrder(bs, marketId, fromTrader, order);
 
     const { settlementTime, publishTime } = await getFastForwardTimestamp(bs, marketId, fromTrader);
     await fastForwardTo(settlementTime, provider());
     const { updateData, updateFee } = await getPythPriceDataByMarketId(bs, marketId, publishTime);
 
-    // Populate a multicall where we settle our own order and try to call mergeAccounts.
-    const calls = [
-      {
-        target: BfpMarketProxy.address,
-        callData: BfpMarketProxy.interface.encodeFunctionData('settleOrder', [
-          fromTrader.accountId,
-          marketId,
-          updateData,
-        ]),
-        value: updateFee,
-        allowFailure: false,
-        requireSuccess: true,
-      },
-      {
-        target: BfpMarketProxy.address,
-        callData: BfpMarketProxy.interface.encodeFunctionData('mergeAccounts', [
-          fromTrader.accountId,
-          toTrader.accountId,
-          market.marketId(),
-        ]),
-        value: bn(0),
-        allowFailure: false,
-        requireSuccess: true,
-      },
-    ];
-
-    const trustedMultiCallForwarder = new ethers.Contract(
-      trustedMulticallerAddress,
-      trustedMulticallerAbi
-    ) as TrustedMulticallForwarder;
-
-    // The trustedMulticaller is not a whitelisted settlement hook. Assert revert.
+    await withExplicitEvmMine(
+      () =>
+        BfpMarketProxy.connect(owner()).setSettlementHookConfiguration({
```

Comment:
Speculate that the recent Cannon had a hidden change that led to `assertRevert` doing unexpected things when trying to parse this error. Either way, the same scenario can be rewritten without the multicall forwarder by disabling deregistering the hook that was used on commitment.